### PR TITLE
Sage knows Hadamard matrices <1201, not 1001

### PR DESCRIPTION
**Pull request by dimpase**
bump the known by Sage Hadamard matrices to 1200, add a reference to the preprint describing the work

### CODE REVIEW

**Comment by dimpase** [comment]
@MatteoCati

File: src/sage/combinat/matrices/hadamard_matrix.py [hunk 1]
```diff
@@ -3055,8 +3059,8 @@ def skew_hadamard_matrix(n, existence=False, skew_normalize=True, check=True):
     Tries to construct a skew Hadamard matrix.
 
     A Hadamard matrix `H` is called skew if `H=S-I`, for `I` the identity matrix
-    and `-S=S^\top`. Currently all orders `\le 1000` for which a construction is
-    known are implemented. For `n > 1000`, only some orders are available.
+    and `-S=S^\top`. Currently all orders `\le 1200` for which a construction is
+    known are implemented. For `n > 1200`, only some orders are available.
```
Comment (dcoudert) [inline src/sage/combinat/matrices/hadamard_matrix.py:3063]: this is correct, but the title of the PR indicates 1201, so may be you should use 1201 here as well ?

**Review by dcoudert — COMMENTED** [review]
otherwise, LGTM.

File: src/sage/combinat/matrices/hadamard_matrix.py (hunk 1 above)
Comment (dimpase) [reply src/sage/combinat/matrices/hadamard_matrix.py:3063]: $<1201$ is the same as $\leq 1200$ 😄 
Sorry.

**Review by dcoudert — APPROVED** [review]
LGTM.

**Review by MatteoCati — APPROVED** [review]

**Pull request merged**